.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    pull_request:
+      branches: 
+        - '**'
+
+jobs:
+  build:
+
+    env:
+      APP_ENV: test
+    
+    strategy:
+      matrix:
+        php: [ '7.3', '7.4' ]
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    
+    - name: Set up PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{matrix.php}}
+        coverage: none
+
+    - name: Install wikimedia-ocr
+      run: |
+        composer install
+        npm run build
```

Comment:
We do have a `build` script in our package.json, but it just compiles assets. Right now we only have PHP CodeSniffer tests which does not depend on assets being compiled, so CI doesn't need to be concerned with the Node dependency at all, I think. But even if it did, I assume you'd need to run `npm install` before running `npm run build`.

Reply:
I just deleted it for simplicity :)